documentation/modules/adding-source-provider.adoc

```diff
@@ -164,8 +241,28 @@ ifdef::ostack[]
 +
 include::snip-certificate-options.adoc[]
 endif::[]
+
 ifdef::cnv[]
-. Click *{virt}*.
+. Access the *Create provider* interface {virt} by doing one of the following:
+
+.. In the {ocp} web console, click *Migration for Virtualization > Providers*.
+... Click *Create Provider*.
+... Select a *Project* from the list. The default project shown depends on the active project of {project-short}. 
++
+If the active project is *All projects*, then the default project is `openshift-mtv`. Otherwise, the default project is the sane as the active project.
```

Comment:
```suggestion
If the active project is *All projects*, then the default project is `openshift-mtv`. Otherwise, the default project is the same as the active project.
```